core/domain/question_services.py

```diff
@@ -117,6 +121,23 @@ def get_question_by_id(question_id):
         return None
 
 
+def get_questions_by_ids(question_ids):
+    """Returns a list of domain objects representing questions.
+
+    Args:
+        question_ids: list(str). List of question ids.
+
+    Returns:
+        list(Question) or None. A list of domain objects representing questions
```

Comment:
list(Question|None)? I mean, None is an alternative to each element of the list, not the entire list ... right?

Reply:
Updated.